README.md

```diff
@@ -31,6 +31,9 @@ Pipelines are **Typed**:
 - [Read about how to use it](/docs/using.md)
 - Look at [some examples](/examples)
 
+_See [our API compatibility policy](api_compatiblity_policy.md) for info on
+the stability level of the API._
+
```

Comment:
there is a typo in the name (url) of the page `api_compatiblity_policy.md` resulting in a `404`

Reply:
haha thanks for catching this, i couldn't even see it after you pointed it out XD